app/persistence/status/status.go

```diff
@@ -1,5 +1,63 @@
 package status
 
+import (
+	"errors"
+	"github.com/hashgraph/hedera-sdk-go"
+	log "github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+	"strconv"
+	"time"
+)
+
 // This table will contain information for latest status of the application
 type Status struct {
+	Name      string
+	AccountID string
+	Code      string
+	Value     string
```

Comment:
`Value` is not that self-explanatory. 
Personally, I think that the DB should have
two columns - `account_id` and `last_fetched_timestamp`
which are `string` and `int64`

Are we going to store something else in there other than the last fetched timestamp for an address?